csharp/test/Ice/optional/TestI.cs

```diff
@@ -28,7 +27,8 @@ public void OpBasicIn(int i1, int? i2, string? i3, string i4, Current current)
             TestHelper.Assert(i3 == null || i3 == i4);
         }
 
-        public (int? R1, int R2, int? R3, string? R4) OpBasicInOut(int i1, int? i2, string? i3,
+        public (int? R1, int R2, int? R3, string? R4) OpBasicInReturnTuple(int i1, int? i2, string? i3,
```

Comment:
Just for reference, you can see that this is where you renamed the method from `OpBasicInOut` to `OpBasicInReturnTuple`